src/cpp/session/SessionOptions.cpp

```diff
@@ -274,7 +274,7 @@ core::ProgramStatus Options::read(int argc, char * const argv[])
          value<bool>(&autoReloadSource_)->default_value(false),
          "Reload R source if it changes during the session")
       ("r-compatible-graphics-engine-version",
-         value<int>(&rCompatibleGraphicsEngineVersion_)->default_value(11),
+         value<int>(&rCompatibleGraphicsEngineVersion_)->default_value(12),
```

Comment:
It looks like we set the compatible graphics engine version in a bunch of places, so we should sanity check that these changes make sense. (I'm a little confused because the versions declared in the various places are different)